talend-component-maven-plugin/src/it/web/.jenkins/scripts/server-registry-stop.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+#  Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# set -xe
+
+# Stop a running server in registry mode
+# Parameters:
+[ -z ${1+x} ] && usage "Parameter 'install_dir'"
+
+INSTALL_DIR="${1}"
+
+# Constants
+DISTRIBUTION_DIR="${INSTALL_DIR}/component-server-distribution"
+
```

Comment:
Please use lower case variables inside the script to use local variables instead of environment variables. Unless it is required ?
Same remark for other variables used afterwards.

Reply:
I used UPPER for global and constant
Constant will always start with "_"